cmd/atrutil/atrutil_main.go

```diff
@@ -538,6 +545,13 @@ func GenerateCriteria(tid string) {
 					continue
 				}
 
+				if !gUnsafe {
+					if unsafeRegex.MatchString(com) {
+						s += "!!!\n"
+						s += "FYI,Potentially destructive command found: " + com
```

Comment:
you can put on one line:
```go
   s += "!!!,Potentially destructive command found: " + com
```

Reply:
I saw that FYI lines are printed to the user when the test was ran, so that's why I did this. Anyway, just made the change.